groups/sig-cluster-lifecycle/groups.yaml

```diff
@@ -145,9 +145,8 @@ groups:
     members:
       - cecilerobertm@gmail.com
       - davanum@gmail.com
-      - detiber@gmail.com
       - fpandini@vmware.com
-      - goldsteina@vmware.com
+      - sbueringer@gmail.com
       - prignanov@vmware.com
```

Comment:
This group should give access to the CAPI staging GCS bucket.

@enxebre should I add you here too and with which email? (assuming it's fine to add myself)

@CecileRobertMichon @vincepri @fabriziopandini To confirm that this change is fine (including the cleanup)

Reply:
(btw I linked all PRs related to maintainer stuff to my maintainer promotion PR, so we have a full list of what we have to do there)

Reply:
lgtm

Reply:
thanks @sbueringer albertogarcial hotmail.com